content/en/docs/setup/upgrade/helm/index.md

```diff
@@ -37,6 +37,20 @@ To get started, check out https://istio.io/latest/docs/setup/getting-started/
 
 {{< boilerplate helm-backup >}}
 
+### Upgrading Custom Resource Definitions (CRDs)
+
+{{< warning >}}
+Helm does not [upgrade or delete](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations)
+CRDs when performing an upgrade. Given this restriction, additional measures are required to ensure that
+Istio is up to date when performing an upgrade.
+{{< /warning >}}
+
+Istio ships CRDs with its helm charts; they are not templated and are found in the base chart under
+a `crds` folder. However, upgrading Istio with Helm requires a change in workflow different from other
+charts due to the required CRDs be updated for proper functionality.
```

Comment:
```suggestion
the `crds` folder. Upgrading Istio with Helm requires a change in workflow due to the required updating of CRDs for proper functionality.
```

Reply:
This feels like too much information; what more is there to say other than what was in the warning?